app/assets/stylesheets/main_controller.scss

```diff
@@ -0,0 +1,104 @@
+.top-section {
+  align-items: center;
+  border-top: solid 10px $orange;
+  justify-content: center;
+  padding: 40px 0;
+
+ .space-bar {
```

Comment:
I would name this separator-line, vertical-separator, vertical-line or something that gets easily associated with what it looks like, space bar makes me think of my keyboard instead of that line :stuck_out_tongue_closed_eyes: